### PR TITLE
feat(sampling): Feature flag for demo purpose - [TET-442]

### PR DESCRIPTION
**Pull request by priscilawebdev**
**What this PR does:**

- Create a new feature flag that will be used for product demo purposes

### CODE REVIEW

File: src/sentry/features/__init__.py [hunk 1]
```diff
@@ -175,6 +175,7 @@
 default_manager.add("organizations:data-forwarding", OrganizationFeature)
 default_manager.add("organizations:discover-basic", OrganizationFeature)
 default_manager.add("organizations:discover-query", OrganizationFeature)
+default_manager.add("organizations:dynamic-sampling-demo", OrganizationFeature, True)
```
Comment (ahmedetefy) [inline src/sentry/features/__init__.py:178]: Do we want the default to be True?

Comment (priscilawebdev) [reply src/sentry/features/__init__.py:178]: yes so we can control it via Flagr

Comment (ahmedetefy) [reply src/sentry/features/__init__.py:178]: got you thanks, where is that doc?

**Review by ahmedetefy — APPROVED** [review]

File: src/sentry/features/__init__.py (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/features/__init__.py:178]: I just updated my comment

**Pull request merged**